### ISSUE TITLE
Buildkit not booting with configuration file: error: dial unix /run/buildkit/buildkitd.sock: connect: no such file or directory

### ISSUE THREAD

**Issue by ramirahikkala** [docker/buildx#289]
What I'm trying to achieve is to get the contents of a directory from a separate image to image I'm creating. I'm having custom registry running on 192.168.1.215:5000 and to get the directory I have this line on Dockerfile:
`COPY --from=192.168.1.215:5000/gaia_ui:1.0 /usr/src/app/build /ui/build`

Obviously, when building I get `http: server gave HTTP response to HTTPS client` error. To overcome the issue I'm trying to create builder with a configuration file with these commands:
```
docker buildx create --config /home/rami/workspace/gaia/buildkitd.toml  --name myconfbuilder4
docker buildx use myconfbuilder4
```

But now if I run `docker buildx inspect --bootstrap`, I get this error:

```
rami@ubuntu:~/workspace/gaia (dockerize)$ docker buildx inspect --bootstrap
[+] Building 7.5s (1/1) FINISHED                                                                                                  
 => ERROR [internal] booting buildkit                                                                                        7.5s
 => => pulling image moby/buildkit:buildx-stable-1                                                                           2.9s
 => => creating container buildx_buildkit_myconfbuilder40                                                                    4.6s
------
 > [internal] booting buildkit:
------
error: dial unix /run/buildkit/buildkitd.sock: connect: no such file or directory
Name:   myconfbuilder4
Driver: docker-container

Nodes:
Name:     myconfbuilder40
Endpoint: unix:///var/run/docker.sock
Error:    listing workers: failed to list workers: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: timed out waiting for server handshake
```

I have tried with your example configuration, with empty configuration and several other contents of the configuration file. Always the same error. Without `--conf` it works, but of course then I have the HTTPS issue. 

My environment:
```
rami@ubuntu:~/workspace/gaia (dockerize)$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic

rami@ubuntu:~/workspace/gaia (dockerize)$ docker version
Client: Docker Engine - Community
 Version:           19.03.8
 API version:       1.40
 Go version:        go1.12.17
 Git commit:        afacb8b7f0
 Built:             Wed Mar 11 01:25:46 2020
 OS/Arch:           linux/amd64
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          19.03.8
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.17
  Git commit:       afacb8b7f0
  Built:            Wed Mar 11 01:24:19 2020
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

rami@ubuntu:~/workspace/gaia (dockerize)$ docker buildx version
github.com/docker/buildx v0.3.1-tp-docker 6db68d029599c6710a32aa7adcba8e5a344795a7
```

**Comment by tonistiigi** [comment]
You have not posted the config that causes an error for you. For accessing your local registry you probably need to run the builder in host network for it to see it https://github.com/docker/buildx#--driver-opt-options

**Comment by ramirahikkala** [comment]
Thanks! 
I tried to add `--driver-opt network=host` but it didn't change anything. I think it is already able to connect to local registry since it's complaining about https.  

I tried several configurations. First with the one from [buildkitd.toml.md](https://github.com/moby/buildkit/blob/master/docs/buildkitd.toml.md) without any changes, one with just `debug = true` line and with an empty configuration. Always the same result. Is there something that should be there on the configuration to make it run (which I am now missing)?
